cpp/harvest/nacjd.cc

```diff
@@ -153,15 +153,15 @@ void ExtractExistingIDsFromMarc(MARC::Reader * const marc_reader, std::set<std::
             StringUtil::ReplaceString("[ICPSR]", "", &id_035);
             StringUtil::TrimWhite(&id_035);
             parsed_marc_ids->emplace(id_035);
-        }
+        }            
```

Comment:
This whitespace change looks strange... i would suppose that our pre-commit hook should automatically use clang-format to normalize this, but for some reason it failed. I will investigate.